db/deploy/computed_columns/application_gis_data.sql

```diff
@@ -2,13 +2,14 @@
 
 begin;
 
-create or replace function ccbc_public.application_gis_data(application ccbc_public.application) 
-returns ccbc_public.gis_data_item as
+drop function ccbc_public.application_gis_data(ccbc_public.application);
+
+create or replace function ccbc_public.application_gis_data(application ccbc_public.application)
```

Comment:
Just wondering if we should rename it to ‘application_gis_latest_data’ to avoid conflicts with the table with same name?

Reply:
Is there conflict? `gis_data` is separate from application, this is the `gis_data` for the application.